app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if not visitor.get("vaccine", 0):

```

Comment:
Instead of using `visitor.get("vaccine", 0)`, you can directly check for the presence of the 'vaccine' key using `if "vaccine" not in visitor:`. This aligns with the checklist's recommendation to avoid unnecessary comparisons to `0`.